.env.dist

```diff
@@ -0,0 +1,2 @@
+export YAHOO_OAUTH_KEY=''
+export YAHOO_OAUTH_SECRET=''
```

Comment:
never mind, just saw in the readme that .env is the for reals name. maybe add a .gitignore with that in it, though